recipes/sdl_ttf/all/conanfile.py

```diff
@@ -59,10 +63,9 @@ def source(self):
                   destination=self._source_subfolder, strip_root=True)
 
     def _patch_sources(self):
-        # Handle sdl2 static target
-        if not self.options["sdl"].shared:
-            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                                  "SDL2::SDL2", "SDL2::SDL2-static")
```

Comment:
Don't forget the patch for 2.0.15 as well because you have removed this block.

Reply:
Sorry. I forget it.
I create patch for 2.0.15.